### PR TITLE
Update dynamic-theme-fixes.config

### PR DESCRIPTION
**Pull request by teorems**
added mix blend mode and font color scheme for code snippets in campus.datacamp.com to improve readabiliy, which previously was mediocre.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4186,6 +4186,21 @@ INVERT
...
 
+campus.datacamp.com
+
+CSS
+.css-hu6jey code {
+     mix-blend-mode: normal !important;
+     color: var(--darkreader-neutral-text) !important;
+     
+}
+.css-hu6jey pre {
```
Comment (Myshor) [inline src/config/dynamic-theme-fixes.config:4197]: Can you share specyfic URL to the course where pre is needed to be fixed too?
I used https://campus.datacamp.com/courses/intro-to-python-for-data-science/chapter-1-python-basics?ex=3 and without this class it worked fine with only .css-hu6jey code fix.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -10567,16 +10582,6 @@ img[src*="bookmark.png"]
 
 ================================
 
-goproblems.com
```
Comment (Myshor) [inline src/config/dynamic-theme-fixes.config:10570]: Removing other fixes won't be accepted to merge for sure!

**Review by Myshor — CHANGES_REQUESTED** [review]
Check my comments.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (teorems) [reply src/config/dynamic-theme-fixes.config:10570]: That was unintended for sure!

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (teorems) [reply src/config/dynamic-theme-fixes.config:4197]: you can check the difference here :

https://campus.datacamp.com/courses/text-mining-with-bag-of-words-in-r/jumping-into-text-mining-with-bag-of-words?ex=3

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -4186,6 +4186,21 @@ INVERT
 
 ================================
 
+campus.datacamp.com
+
+CSS
+.css-hu6jey code {
+     mix-blend-mode: normal !important;
+     color: var(--darkreader-neutral-text) !important;
+
```
[the remark this answers is not in the document]
Comment (teorems) [reply src/config/dynamic-theme-fixes.config:4195]: Yeah this is the same code in datacamp.com, which doesn't apply for the time being in campus.datacamp . No specific reason for the whitespace

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -4186,6 +4186,21 @@ INVERT
 
 ================================
 
+campus.datacamp.com
```
[the remark this answers is not in the document]
Comment (teorems) [reply src/config/dynamic-theme-fixes.config:4189]: I think this fix is important as all the code snippets are barely readable in dark mode . I agree in principle with the alias, but i don't have time to test it. We should check if the pre fix works for the video sequences .

**Comment by Myshor** [comment]
@teorems Ok... Checked all replys.
Confirm that additional selector is needed.

Checked how video would work with merging all into bandcamp.com + *.bandcamp.com alias and to make it faster I will merge it on your branch. 😉

**Review by Myshor — APPROVED** [review]

**Pull request merged**